src/test/suite/recorded.test.ts

```diff
@@ -43,7 +41,6 @@ suite("recorded test cases", async function () {
     "../../../src/test/suite/fixtures/recorded"
   );
   const files = walkFilesSync(directory);
-  enableDebugLog(false);
```

Comment:
So during tests, `context.extensionMode !== ExtensionMode.Development`?

Reply:
Correct. There are three modes: production, development and test